js/app/modules/card/thumb.js

```diff
@@ -166,7 +166,7 @@ const Thumb = new Module.Class({
             this.set_label_or_hide(this._synopsis_label, this.model.synopsis);
             this._title_label.valign = Gtk.Align.END;
         } else {
-            this._title_label.valign = Gtk.Align.CENTER;
+            this._title_label.valign = Gtk.Align.START;
```

Comment:
ya. this only makes a difference for horizontally orientated thumb cards, which as far as I can see, never actually show up in any existing apps